crates/router/src/connector/volt/transformers.rs

```diff
@@ -405,6 +450,68 @@ impl TryFrom<types::RefundsResponseRouterData<api::Execute, RefundResponse>>
     }
 }
 
+#[derive(Debug, Deserialize, Clone, Serialize)]
+pub struct VoltWebhookBodyReference {
+    pub payment: String,
+    pub merchant_internal_reference: Option<String>,
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct VoltWebhookBodyEventType {
+    pub status: VoltWebhookStatus,
+    pub detailed_status: Option<VoltDetailedStatus>,
+}
+
+#[derive(Debug, Deserialize, Clone, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct VoltWebhookObjectResource {
+    pub reference: String,
```

Comment:
Check how this is being populated, if not required remove this

Reply:
As per connector response this will be generated by volt if unique reference field is not populated by us, So remove this field from the struct and use merchant_internal_reference.unwrap_or(payment) as replacement